source/SIL.AppBuilder.Portal.Frontend/src/ui/components/loaders/overlay-loader.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+import RectLoader from './rect-loader';
+
+export default function OverlayLoader(props: any) {
+  return (
+    <div
+      data-test-overlay-loader
+      className='flex-row flex-grow justify-content-conter align-items-center'
+      {...props}
+      style={{
+        opacity: 0.5,
```

Comment:
inline styles are cool in react, right?

these aren't reusable, so here they be

Reply:
maybe they would be if we had a legit style compiler though.

one day

Reply:
They still clutter the jsx, and are meaningless unless you are styling. Which is why I keep them in a local stylesheet as a sibling. :)

Reply:
I will let it pass THIS time ;)

Reply:
they do. This was *pure* laziness